notebooks/network_mapping/Supervised_Asset_Classification.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>Say a bit more about the process behind the categorization. What's the intent, and how are they being categorized.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>